examples/nlp_seq/train.py

```diff
@@ -296,17 +300,29 @@ def initialize_variables(init_rng):
     return init_variables
   init_variables = initialize_variables(init_rng)
 
-  optimizer_def = optim.Adam(learning_rate, beta1=0.9, beta2=0.98,
-      eps=1e-9, weight_decay=1e-1)
-  optimizer = optimizer_def.create(init_variables['params'])
-  optimizer = jax_utils.replicate(optimizer)
-
   learning_rate_fn = create_learning_rate_scheduler(
       base_learning_rate=learning_rate)
 
+  optimizer = optax.adamw(
+      learning_rate_fn, b1=0.9, b2=0.98, eps=1e-9,
+      weight_decay=1e-1
+      )
+  state = train_state.TrainState.create(
+    apply_fn=model.apply,
```

Comment:
nit: we usually indent with 4 spaces for function args

Reply:
done. Thanks.